packages/paste-website/src/pages/components/form-pill-group/index.mdx

```diff
@@ -124,88 +142,183 @@ The Form Pill Group is focusable, but only one pill is focusable at a time. This
   </AspectRatio>
 </Box>
 
-#### Display Pill vs. Form Pill
+#### Form Pill vs. Display Pill
+
+Form Pill Group creates a list from which a user may select items, whereas a Display Pill Group creates a list of static items.
 
-There are some very important semantic differences between a Display Pill and Form Pill Group. Below is a table explaining the main differences to get a better understanding of when to use each type of pill.
+Use the table below to get a better understanding of when to use Form Pill or Display Pill.
 
 <FormPillVsDisplayPillTable />
 
 ## Examples
 
-### Basic Form Pill Group
+### Basic
+
+This Form Pill example shows the basic static component that's exported by Paste. A Form Pill can have an optional Avatar or Icon placed before the text.
 
-The basic Form Pill Group can be composed by adding `FormPill` components as children of `FormPillGroup`. These must be used with the `useFormPillState` hook, and the returned state should be spread onto each component.
+Interaction states on Form Pills need to be managed separately as shown in the examples after this one by using the `useFormPillState` hook. The returned state should be spread onto each component. This will provide you with some internal state management and keyboard navigation.
```

Comment:
```suggestion
Interaction states on Form Pills need to be managed separately as shown in the following examples by using the `useFormPillState` hook. The returned state should be spread onto each component. This will provide you with some internal state management and keyboard navigation.
```

Suggestion(non-blocking): using "the examples after this one" made me think that this example wouldn't use it, but it does